config/defaults.php

```diff
@@ -24,7 +24,7 @@
             ],
         ],
         'timezoneAwareDateTimeWidget' => false,
-        'useAssetCompress' => Plugin::loaded('AssetCompress'),
+        'useAssetCompress' => Plugin::isLoaded('AssetCompress'),
```

Comment:
https://github.com/FriendsOfCake/crud-view/blob/master/composer.json#L32
needs to be at least 3.7 for `isLoaded`